examples/assets/video/city_halfres_hls.m3u8

```diff
@@ -0,0 +1,9 @@
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:8
+#EXT-X-MEDIA-SEQUENCE:0
+#EXTINF:7.166667,
+city_halfres_hls0.ts
+#EXTINF:0.866667,
+city_halfres_hls1.ts
```

Comment:
does this indicate an audio track (albeit a possibly empty one)?

Reply:
Those aren't separate tracks, those are separate segments; if you look with VLC for example you can see there is no audio